plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/cookiecutter.ts

```diff
@@ -86,12 +100,18 @@ export function createFetchCookiecutterAction(options: {
 
       const cookiecutter = templaters.get('cookiecutter');
 
+      const values = {
+        ...ctx.input.values,
+        copyWithoutRender: ctx.input.copyWithoutRender,
+        extensions: ctx.input.extensions,
+      };
+
       // Will execute the template in ./template and put the result in ./result
       await cookiecutter.run({
         workspacePath: workDir,
         dockerClient,
         logStream: ctx.logStream,
-        values: ctx.input.values as TemplaterValues,
+        values: values as TemplaterValues,
```

Comment:
I think you want the `as TemplaterValues` to be moved up to L104 here - that's the unknown type that needs coercion (ctx.input I think comes out as `any`), although doing it here does still typecheck due to the definition of `TemplaterValues`.